### DISCUSSION TITLE
yt3.ggpht.com

### DISCUSSION THREAD

**Discussion by YBS-PC** [AdguardTeam/AdGuardHome#5592] [Q&A]
I don't see static content on youtube (avatars, etc.) - yt3.ggpht.com blocked. How can I set up redirection or something else so that these pictures are displayed. I tried DNS rewrites and pointed in the from field yt3.ggpht.com , and in the to field specified yt4.ggpht.com - it didn't help, and there are no requests in the request log yt4.ggpht.com.

**Comment by YBS-PC** [comment]
And is it possible to apply DNS rewrites to a specific client, but without applying blocking filters rules ?

**Comment by perdakovich** [comment]
I would like to know about this too
